lib/ansible/modules/crypto/openssh_cert.py

```diff
@@ -0,0 +1,544 @@
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: openssh_cert
+author: "David Kainz (@lolcube)"
+version_added: "2.8"
+short_description: Generate OpenSSH host or user certificates.
+description:
+    - Generate and regenerate OpensSSH host or user certificates.
+requirements:
+    - "ssh-keygen"
+options:
+    state:
+        required: false
+        default: "present"
+        choices: [ 'present', 'absent' ]
+        description:
+            - Whether the host or user certificate should exist or not, taking action if the state is different from what is stated.
+    type:
+        required: true
+        choices: ['host', 'user']
+        description:
+            - Whether the module should generate a host or a user certificate.
+    force:
+        required: false
+        default: false
+        type: bool
+        description:
+            - Should the certificate be regenerated even if it already exists and is valid.
+    path:
+        required: true
+        type: path
+        description:
+            - Path of the file containing the certificate.
+    signing_key:
+        required: true
+        type: path
+        description:
+            - The path to the private openssh key that is used for signing the public key in order to generate the certificate.
+    public_key:
+        required: true
+        type: path
+        description:
+            - The path to the public key that will be signed with the signing key in order to generate the certificate.
+    valid_from:
+        required: true
+        type: str
+        description:
+            - "The point in time the certificate is valid from. Time can be specified either as relative time or as absolute timestamp.
+               Time will always be interpreted as UTC. Valid formats are: C([+-]timespec | YYYY:MM:DD | YYYY:MM:DD:HH:MM:SS | YYYY:MM:DD HH:MM:SS | always)
+               where timespec can be an integer + C([w | d | h | m | s]) (e.g. C(+32w1d2h).
+               Note that if using relative time this module is NOT idempotent."
+    valid_to:
+        required: true
+        type: str
+        description:
+            - "The point in time the certificate is valid to. Time can be specified either as relative time or as absolute timestamp.
+               Time will always be interpreted as UTC. Valid formats are: C([+-]timespec | YYYY:MM:DD | YYYY:MM:DD:HH:MM:SS | YYYY:MM:DD HH:MM:SS | forever)
+               where timespec can be an integer + C([w | d | h | m | s]) (e.g. C(+32w1d2h).
+               Note that if using relative time this module is NOT idempotent."
+    valid_at:
+        required: false
+        type: str
+        description:
+            - "Check if the certificate is valid at a certain point in time. If it is not the certificate will be regenerated.
+               Time will always be interpreted as UTC. Mainly to be used with relative timespec for C(valid_from) and / or C(valid_to).
```

Comment:
[Ansible's developer documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation) says `I(...)` should be used for options.
```suggestion
               Time will always be interpreted as UTC. Mainly to be used with relative timespec for I(valid_from) and / or I(valid_to).
```